onward/app/weather/models/CityResponse.scala

```diff
@@ -2,11 +2,21 @@ package weather.models
 
 import common.Edition
 import common.editions.{Au, Uk, Us}
+import play.api.libs.json.{Json, Writes}
 import weather.models.accuweather.LocationResponse
-import play.api.libs.json.Json
 
 object CityResponse {
-  implicit val jsonWrites = Json.writes[CityResponse]
+  implicit val jsonReads = Json.reads[CityResponse]
+
+  implicit val writes = new Writes[CityResponse] {
+    def writes(model: CityResponse) = {
```

Comment:
Trying to work out why the default `writes` wouldn't work? 
```suggestion
    def writes(model: CityResponse) = Json.writes[CityResponse]
```

Reply:
oh maybe it would! I just went and added these everywhere 😅 if not needed happy to remove